packages/sw-background-sync-queue/src/lib/background-sync-idb-helper.js

```diff
@@ -0,0 +1,42 @@
+import IDBHelper from '../../../../lib/idb-helper';
```

Comment:
@jeffposnick this is fixed in https://github.com/GoogleChrome/sw-helpers/pull/72/commits/b6bdf7b9f3bca965beeecdb16866464a47ca0882
can you please have a quick look?